samples/wifi/provisioning/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(wifi_ble_provisioning)
+
+set(WIFI_NRF_DRV_BASE ${WEST_TOPDIR}/nrf/drivers/wifi/nrf700x)
+if(CONFIG_WPA_SUPP)
+set(WPA_SUPP_BASE ${WEST_TOPDIR}/modules/lib/hostap)
+endif()
```

Comment:
similar comment as the `WIFI_NRF_DRV_BASE`.

This variable doesn't appear top be used anywhere.

Reply:
I use wifi shell example as template, where these variables are defined. It seems the build will not fail if I remove them. @krish2718 Can you help to confirm if it is fine to remove these two variables?

Reply:
Yes, this is a leftover from cleanup, both the variables can be removed.

Reply:
Thanks. I will remove both.